safety/scan/command.py

```diff
@@ -620,12 +623,15 @@ def sort_vulns_by_score(vuln: Vulnerability) -> int:
         if not no_output:
             console.print("-" * console.size.width)
 
+    run_easter_egg(console, exit_code)
```

Comment:
I adjusted it to only run if the output format is SCREEN

Reply:
feature flag- so the code is currently behind an environment variable: `SAFETY_FUN_MODE`
The easter egg code will only run if `SAFETY_FUN_MODE` is set and set to an acceptable value (`allowed_modes = {"ascii", "fx", "race", "fortune"}`)

I thought about having it be a feature flag (ie flag in the scan command), but I thin kit works better with this env setup. As a feature flag it would mean more adjustments int he `scan` function vs just having 2lines there now. 
I can't imagine a user accidentally setting `SAFETY_FUN_MODE` as an env variable to one of our allowed types in any case other than intentionally done. Let me know what you think!

Reply:
@jakub-safetycli Are there any other ways you are thinking for making a user opt in other than having to set the ENV Var?

Reply:
I think the env variable is fine in this case.